layouts/partials/docs/search.html

```diff
@@ -1,9 +1,15 @@
-<form class="relative" action="{{ relref . "/docs/search" }}" method="GET">
-    <i class="fas fa-search absolute inset-y-0 flex items-center pl-3 text-gray-500"></i>
+<script type="text/javascript">
```

Comment:
Nit: No need to specify `type="text/javascript"`:

```suggestion
<script>
```